src/table/utils/keyboard-utils.ts

```diff
@@ -15,6 +15,9 @@ export const isArrowKey = (key: string) =>
 
 export const isShiftArrow = (evt: React.KeyboardEvent) => evt.shiftKey && isArrowKey(evt.key);
 
+export const isFirstHeaderCell = (coord: [number, number]) =>
+  coord[0] === FIRST_HEADER_CELL_COORD[0] && coord[1] === FIRST_HEADER_CELL_COORD[1];
+
```

Comment:
This type of function should live in accessibility-utils

Reply:
with new changes, this function is not needed